docker-run

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+
+run_docker () {
+    docker run -it --rm -v $(pwd)/generated_docs:/work/streisand/generated_docs \
```

Comment:
Can you have this script also check that `docker` is a command in the `$PATH` and if not indicate that you need to install docker first?

Reply:
I started doing this with `-z "$(type -f docker 2>/dev/null)"` but turned it into an `if ! docker 2>/dev/null` instead; any preference?

Reply:
I think the first might be more idiomatic for a "command is installed" check and is what we do in [the util/ansible_check.sh script](https://github.com/StreisandEffect/streisand/blob/master/util/ansible_check.sh#L11).

Reply:
OK, I'm planning on making a util/docker_check.sh, because besides existence and version, there are probably some other things I can do for sanity checking.